src/models/pipeline-run-task-step/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as PropTypes from 'prop-types';
+
+import { ProgressStatus } from '../progress-status';
+
+export interface PipelineRunTaskStepModel {
+  name: string;
+  started?: Date;
+  ended?: Date;
+  status?: ProgressStatus;
```

Comment:
`status` is marked with isRequired in PropTypes declaration, but is optional in interface.
from previous experience I assume it should be required in the interface as well.

```suggestion
  status: ProgressStatus;
```